### PR TITLE
feat(storage): modify docs for configuring storage using env vars

### PR DESCRIPTION
**Pull request by smothiki**

### CODE REVIEW

**Comment by smothiki** [comment]
https://gist.github.com/smothiki/67ab26fbca334e421aefc5dbc8fd51bb/adba6008956c9a74d305b63b2394c9909b71dc54

File: src/installing-workflow/configuring-object-storage.md [hunk 1]
```diff
@@ -18,12 +18,26 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
     * Note: If you provide credentials with sufficient access to the underlying storage, Workflow components will create the buckets if they do not exist.
 * **Step 2:** If applicable, generate credentials that have write access to the storage buckets created in Step 1
 * **Step 3:** If you haven't already fetched the helm classic chart, do so with `helmc fetch deis/workflow-beta3`
-* **Step 4:** Open the helm classic chart with `helmc edit workflow-beta3` and look for the template file `tpl/generate_params.toml`
-* **Step 5:** Update the `storage` parameter to reference the storage platform you are using: `s3`, `azure`, `gcs`
-* **Step 6:** Update the values in the section which corresponds to your storage type, including region, bucket names and access credentials
+* **Step 4:** You can update storage details to your deis/workflow-beta3 chart either by using environment variables or by modifying template file `tpl/generate_params.toml`
+    * **1.** Using environment variables:
+        * set `STORAGE_TYPE` to `s3`, `azure`, `gcs` and set the below environment variables accordingly
```
Comment (bacongobbler) [inline src/installing-workflow/configuring-object-storage.md]: this almost seems too long to be in a bulet point form, though I'm not sure how to resolve this.

File: src/installing-workflow/configuring-object-storage.md [hunk 2]
```diff
@@ -18,12 +18,26 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
...
+            For STORAGE_TYPE=gcs
+            export GCS_KEY_JSON, GCS_REGISTRY_BUCKET, GCS_DATABASE_BUCKET, GCS_BUILDER_BUCKET
+
+            For STORAGE_TYPE=s3
+            export AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGISTRY_BUCKET, AWS_DATABASE_BUCKET, AWS_BUILDER_BUCKET, S3_REGION
+
+            For STORAGE_TYPE=azure
+            export AZURE_ACCOUNT_NAME, AZURE_ACCOUNT_KEY, AZURE_REGISTRY_CONTAINER, AZURE_DATABASE_CONTAINER, AZURE_BUILDER_CONTAINER
+            ```
+    * **1.** Using template file `tpl/generate_params.toml`:
```
Comment (bacongobbler) [inline src/installing-workflow/configuring-object-storage.md]: should be `2.`

**Comment by bacongobbler** [comment]
> https://gist.github.com/smothiki/67ab26fbca334e421aefc5dbc8fd51bb/adba6008956c9a74d305b63b2394c9909b71dc54

You can just hit the "View" button [when reviewing the document](https://github.com/deis/workflow/pull/240/files). It'll redirect you to [the raw document](https://github.com/smothiki/workflow/blob/36eaacb67b1ef67c5016518c5a04d19d23f6c686/src/installing-workflow/configuring-object-storage.md) for viewing. :)

File: src/installing-workflow/configuring-object-storage.md (hunk 1 above)
Comment (mboersma) [inline src/installing-workflow/configuring-object-storage.md]: It renders ok, although a sub-list might be a better format choice.

* **1.** Using environment variables:
  * set `STORAGE_TYPE` to `s3`, `azure`, `gcs` and set the below environment variables accordingly
    * For `STORAGE_TYPE=gcs`:

      ```
       export GCS_KEY_JSON, GCS_REGISTRY_BUCKET, GCS_DATABASE_BUCKET, GCS_BUILDER_BUCKET
      ```
    * For `STORAGE_TYPE=s3`:

      ```
      export AWS_ACCESS_KEY, AWS_SECRET_KEY, AWS_REGISTRY_BUCKET, AWS_DATABASE_BUCKET, AWS_BUILDER_BUCKET, S3_REGION
      ```

File: src/installing-workflow/configuring-object-storage.md [hunk 3]
```diff
@@ -18,12 +18,34 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
     * Note: If you provide credentials with sufficient access to the underlying storage, Workflow components will create the buckets if they do not exist.
 * **Step 2:** If applicable, generate credentials that have write access to the storage buckets created in Step 1
 * **Step 3:** If you haven't already fetched the helm classic chart, do so with `helmc fetch deis/workflow-beta4`
-* **Step 4:** Open the helm classic chart with `helmc edit workflow-beta4` and look for the template file `tpl/generate_params.toml`
-* **Step 5:** Update the `storage` parameter to reference the storage platform you are using: `s3`, `azure`, `gcs`
-* **Step 6:** Update the values in the section which corresponds to your storage type, including region, bucket names and access credentials
+* **Step 4:** You can update storage details to your `deis/workflow-beta4` chart either by using environment variables or by modifying template file `tpl/generate_params.toml`
```
Comment (mboersma) [inline src/installing-workflow/configuring-object-storage.md]: Maybe reword this a bit:
> Update storage details either by setting the appropriate environment variables or by modifying the template file `tpl/generate_params.toml`

File: src/installing-workflow/configuring-object-storage.md [hunk 4]
```diff
@@ -18,12 +18,34 @@ The helm classic chart for Deis Workflow can be easily configured to connect Wor
...
+    * **1.** Using environment variables:
+        * set `STORAGE_TYPE` to `s3`, `azure`, `gcs` and set the following environment variables accordingly
```
Comment (mboersma) [inline src/installing-workflow/configuring-object-storage.md]: There should be an "or" before `gcs`. Maybe reword it this way:
> Set `STORAGE_TYPE` to `s3`, `azure`, or `gcs`, then set the following environment variables accordingly.
